Applications/Spire/Include/Spire/Ui/TabWidget.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef SPIRE_TAB_WIDGET_HPP
+#define SPIRE_TAB_WIDGET_HPP
+#include <QTabWidget>
```

Comment:
Include the forward declared file "Spire/Ui/Ui.hpp" and forward declare this class.

Reply:
Previously, in a different PR I believe, you mentioned we were moving away from using forward declarations in headers (like Ui.hpp). Was that a specific case? Should I still limit the amount of #includes I have in headers?

Reply:
Nope, what I said is that forward declarations are outdated and generally suck but we still use them since they are a cheap way to provide IDE support/autocompletion.

Reply:
Okay, fair.